apps/entity-service/src/entities/entities.service.ts

```diff
@@ -157,7 +159,14 @@ export class EntitiesService {
   fullUrl = (media: Media) => this.mediaService.getUrl(media);
   thumbnailUrl = (media: Media) => this.mediaService.getUrl(media, "thumbnail");
 
-  mediaDto = (media: Media) => new MediaDto(media, this.fullUrl(media), this.thumbnailUrl(media));
+  mediaDto(media: Media) {
+    return new MediaDto(media, {
+      url: this.fullUrl(media),
+      thumbUrl: this.thumbnailUrl(media),
+      entityType: media.modelType as EntityType,
+      entityUuid: media.modelId.toString()
```

Comment:
A couple things I'm confused about here: 
* The DTO doesn't show there being these `entity` fields, nor does it seem to have entity UUID at all
* `modelType` is _not_ an `EntityType`, it's a Laravel model string (see my comment above about wanting to map those to an entity type
* `modelId` is not a UUID, it's a numerical int ID

Reply:
changed